packages/authentication/utils/login.ts

```diff
@@ -13,5 +14,8 @@ export const isLoginChangeExpiredPasswordRedirectResponse = (
   data: LoginResponse,
 ): data is LoginChangeExpiredPasswordRedirectResponse => {
   const redirectUrl: keyof LoginChangeExpiredPasswordRedirectResponse = 'redirectUrl'
-  return redirectUrl in data
+  return (
+    redirectUrl in data ||
+    ((data as AllAuthResponse).data?.user && 'redirectUrl' in (data as AllAuthResponse).data.user)
+  )
```

Comment:
You shouldn't be worrying about this for this story. It will be fixed here `https://silverlogic.atlassian.net/browse/BA-1919`